torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -152,6 +152,22 @@ AOTI_TORCH_EXPORT AOTITorchError aoti_torch_create_tensor_from_blob(
     AtenTensorHandle* ret // returns new reference
 );
 
+AOTI_TORCH_EXPORT AOTITorchError aoti_torch__scaled_dot_product_flash_attention(
+    AtenTensorHandle* ret0, // returns new reference
+    AtenTensorHandle* ret1, // returns new reference
+    AtenTensorHandle* ret2, // returns new reference
+    AtenTensorHandle* ret3, // returns new reference
+    int64_t* ret4,
+    int64_t* ret5,
+    AtenTensorHandle* ret6, // returns new reference
+    AtenTensorHandle* ret7, // returns new reference
+    AtenTensorHandle* ret8, // returns new reference
```

Comment:
> Looks like we have to put return values at the beginning of parameters because of varargs, so we should just revert #109834 for consistency. We can still change at this point.

I will revert #109834 once the variadic solution is accepted.